src/main/kotlin/no/nav/familie/tilbake/behandlingskontroll/BehandlingskontrollService.kt

```diff
@@ -197,7 +208,7 @@ class BehandlingskontrollService(private val behandlingsstegstilstandRepository:
                                                                                         behandlingsstegsinfo.behandlingssteg)
         when (gammelBehandlingsstegstilstand) {
             null -> {
-                settBehandlingsstegOgStatus(behandlingId, behandlingsstegsinfo)
+                opprettBehandlingsstegOgStatus(behandlingId, behandlingsstegsinfo)
```

Comment:
Jeg synes settBehandlingsstegOgStatus høres bedre ut enn opprettBehandlingsstegOgStatus. Vi setter et steg og stegsstatus for en behandling.

Reply:
Men den oppretter jo faktisk forekomster i basen. Alternativene er jo opprett eller oppdater. Alle andre steder så oppfatter man at en setter oppdaterer et felt.

Reply:
skal vi flisespikke så blir egentlig navnet `opprettBehandlingsstegOgOppdaterBehandlingsstatus`